### ISSUE TITLE
Unable to install due to peer dependency conflict

### ISSUE THREAD

**Issue by muter** [Byndyusoft/nest-kafka#66]
Hello!

I am running a project with `class-transformer@0.5.1` among it's dependencies.

I'm trying to install the library, using the usual `npm i --save @byndyusoft/nest-kafka`.

This, however, fails with the following error message:
```
npm ERR! While resolving: 
npm ERR! Found: class-transformer@0.5.1
npm ERR! node_modules/class-transformer
npm ERR!   class-transformer@"0.5.1" from the root project
npm ERR!   peer class-transformer@"^0.5.1" from @byndyusoft/nest-kafka@1.2.1
npm ERR!   node_modules/@byndyusoft/nest-kafka
npm ERR!     @byndyusoft/nest-kafka@"*" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer class-transformer@"^0.4.0" from @byndyusoft/class-validator-extended@1.0.0
npm ERR! node_modules/@byndyusoft/class-validator-extended
npm ERR!   peer @byndyusoft/class-validator-extended@"^1.0.0" from @byndyusoft/nest-kafka@1.2.1
npm ERR!   node_modules/@byndyusoft/nest-kafka
npm ERR!     @byndyusoft/nest-kafka@"*" from the root project
```
Changing version of `class-transformer` to `0.4.0` in root project yields virtually the same result.

As far as I can say, this is caused by conflict between peer-dependency of `class-transformer@^0.5.1`  and transitive peer dependency of `@byndyusoft/class-validator-extended@"^1.0.0` -> `class-transformer@"^0.4.0`.

Please note, that versions `^0.4.0` and `^0.5.1` are not [considered compatible](https://docs.npmjs.com/cli/v6/using-npm/semver#caret-ranges-123-025-004)

Thank you!

* Software versions:
  - `docker image: node:18.8.0-slim`
  - `node`: v18.8.0
  - `npm`: 8.18.0
  - `nestjs`: 8.4.7
  - `@byndyusoft/nest-kafka`: 1.2.1
  - `class-transformer`: 0.5.1 or 0.4.0

**Comment by KillWolfVlad** [comment]
Hello! We have released a new version 1.0.1 `@byndyusoft/class-validator-extended`. Now you can install `@byndyusoft/nest-kafka`?

P.S. We use [Yarn Modern](https://yarnpkg.com) 🧶 because it is the best package manager for node. It would be cool if you use it too!

**Comment by muter** [comment]
Yep, installs as intended now. 

Will report, if met with any other issues.

Thanks for quick response!
Hold on there.
